core/service/src/components.rs

```diff
@@ -145,10 +145,11 @@ pub type PoolApi<C> = <C as Components>::TransactionPoolApi;
 pub trait RuntimeGenesis: Serialize + DeserializeOwned + BuildStorage {}
 impl<T: Serialize + DeserializeOwned + BuildStorage> RuntimeGenesis for T {}
 
-/// Something that can create initial session keys from given seeds.
+/// Something that can create and store initial session keys from given seeds.
 pub trait InitialSessionKeys<C: Components> {
-	/// Generate the initial session keys for the given seeds.
-	fn generate_intial_session_keys(
+	/// Generate the initial session keys for the given seeds and store them in
+	/// an internal keystore.
```

Comment:
```suggestion
	/// the keystore.
```

Otherwise it sounds like just "some" keystore to me.

Reply:
what is "the" keystore?
From the perspective of the trait, which is an _abstract interface_, it should be "some" keystore -- I'm glad it came across correctly.